chapters/connectors.tex

```diff
@@ -246,16 +242,10 @@ \subsection{Expandable Connectors}\doublelabel{expandable-connectors}
 \end{example}
 
 \item
-  expandable connectors can only be connected to other expandable
-  connectors.
+  expandable connectors can only be connected to other expandable connectors.
 \end{itemize}
-  If a connect equation references a potentially present variable, or
-  variable element, in an expandable connector the variable or variable
-  element is marked as being present, and due to the paragraphs above it
-  is possible to deduce whether the bus variable shall be treated as
-  input, or shall be treated as output in the connect equation. That
-  input or output prefix is added if no input/output prefix is present
-  on the declaration
+
+If a connect equation references a potentially present variable, or variable element, in an expandable connector the variable or variable element is marked as being present, and due to the paragraphs above it is possible to deduce whether the bus variable shall be treated as input, or shall be treated as output in the connect equation.  That \lstinline!input! or \lstinline!output! prefix is added if no \lstinline!input!/\lstinline!output! prefix is present on the declaration.
```

Comment:
as far as I understand this is an unrelated formatting change, or is there some other change I'm not seeing.

Reply:
Yes, only cleanup related to expandable connectors in general (this is the `\lstinline` part):
> Improve readability of source of nested item list on expandable connectors
>
> Also throwing in some missing \lstinline.

I agree that `\lstinline` cleanup doesn't really belong here.  It's just that I'm not sure we'll ever actually sit down and fix such formatting for the entire specification at once.  Therefore, I was trying the approach of gradually fixing the situation by improving things where work is done at the moment.  Perhaps we need another strategy…

Reply:
The stuff that doesn't belong to this PR has been extracted to #2606.  Getting that on _master_ and then resolving the conflicts should free this PR of unrelated cleanup.

Reply:
Ok, makes sense.